cegs_portal/search/tsv_templates/v1/non_targeting_reos.py

```diff
@@ -1,11 +1,42 @@
+from cegs_portal.search.templatetags.custom_helpers import format_significance
+
+
 def non_targeting_regulatory_effects(data):
     reo_page, feature = data
```

Comment:
`reo_page` should be renamed to `reos` since the data isn't paginated.

Reply:
Right